packages/flutter/test/material/slider_theme_test.dart

```diff
@@ -590,6 +590,144 @@ void main() {
       )
     );
   });
+
+  // Only the thumb, overlay, and tick mark have special shortcuts to provide
+  // no-op or empty shapes.
+  //
+  // The track can also be skipped by providing 0 height.
+  //
+  // The value indicator can be skipped by passing the appropriate
+  // [ShowValueIndicator].
+  testWidgets('The slider shape paintings can be variously skipped or removed', (WidgetTester tester) async {
```

Comment:
Nit/Optional: I would suggest breaking this into multiple smaller tests to make it easier to follow.

Reply:
Done